docs/services/wetty.md

```diff
@@ -0,0 +1,46 @@
+# Wetty
+
+[Wetty](https://github.com/butlerx/wetty/tree/main) is an SSH terminal over HTTP/HTTPS, useful for when on a strict network which disallows outbound SSH traffic, or when only a browser can be used (like a managed chromebook).
+
+## Dependencies
+
+This service requires the following other services:
+
+- a [Traefik](traefik.md) reverse-proxy server
+
+## Configuration
+
+To enable this service, add the following configuration to your `vars.yml` file and re-run the [installation](../installing.md) process:
+
+```yaml
+########################################################################
+#                                                                      #
+# wetty                                                                #
+#                                                                      #
+########################################################################
+
+wetty_enabled: true
+wetty_hostname: mash.example.com
+wetty_path_prefix: /wetty
+wetty_ssh_host: example.com
+wetty_ssh_port: 22
```

Comment:
I suppose this only works for remote machines? If one wants to connect to the local machine, how can it be done? I suppose `localhost` at port `22` won't work, since that's likely inaccessible from the wetty container.

Reply:
You are correct that localhost will not work since that would be localhost inside the container

To connect to the same machine which is running Wetty I specify its external IP, or its domain which routes to its external IP. 

If SSH is not bound to the external interface a VPN IP (from a tunnel interface) can be used.